### DISCUSSION TITLE
升级7.8.4后需要激活码

### DISCUSSION THREAD

**Discussion by doremiooo** [getlantern/lantern-forum-cn#409] [Q&A]
@Mike777Khoo 之前正常的专业版，经历闪退；在放出7.8.4后，安装了7.8.4，账号变成了普通版，使用邮箱登录提示需要激活码。

**Comment by MaxMKMax** [comment]
@doremiooo 
嗯嗯. 好的. 我会帮您安排. 两三天之后请您留意邮箱或GitHub. 谢谢您支持蓝灯!

**Reply by doremiooo** [reply]
已经激活了，感谢。

**Reply by MaxMKMax** [reply]
@doremiooo 嗯嗯. 好的. 不客气. 谢谢您支持蓝灯. 有什么需要帮忙的随时再找我!
